app/templates/sidebars/settings.hbs

```diff
@@ -1,18 +1,47 @@
-<nav class="layout-sidebar">
-  <ul class='nav sidebar-nav'>
+{{#each session.currentUser.organizations as |organization|}}
+  <ul class="nav sidebar-nav">
     <li class="title">
-      {{session.currentUser.name}} Settings
+      {{organization.name}} Settings
     </li>
-    {{#activating-item currentWhen='settings.profile' tagName='li'}}
-      {{link-to 'Profile' 'settings.profile'}}
-    {{/activating-item}}
+    {{#link-to 'organization.members' organization tagName="li"}}
+      {{link-to "Members" "organization.members" organization bubbles=false}}
+    {{/link-to}}
+    {{#link-to 'organization.roles' organization tagName="li"}}
+      {{link-to "Roles" "organization.roles" organization bubbles=false}}
+    {{/link-to}}
+    {{#link-to 'organization.environments' organization tagName="li"}}
```

Comment:
No, there isn't really a better solution here.  We are working to get `is-active` type helpers in Ember for routing, so you can just do:

```
<li class="{{if (is-active-route "organization.environments" organization) 'active'}}">
  {{! link-to goes here}}
</li>
```

But that is likely at least 2.3/2.4 before landing.